webapp/home/templates/home/snippets/about/about.html

```diff
@@ -36,43 +40,99 @@ <h3 id="acknowledgement-statement">
   operates Galaxy Australia in collaboration with
   <a href="https://www.qcif.edu.au/" target="_blank">QCIF</a>,
   <a href="https://www.unimelb.edu.au/" target="_blank">The University of Melbourne</a>
   and
   <a href="https://www.aarnet.edu.au/" target="_blank">AARNet</a>.
-  The service is underpinned by computational resources provided by
-  <a href="https://www.unimelb.edu.au/" target="_blank">The University of Melbourne</a>,
+</p>
+<div class="container logo">
+    <div class="text-center">
+      <a href="https://www.unimelb.edu.au" target="_blank">
+        <img src="{% static 'logos/footer/unimelb-2.svg' %}" />
+      </a>
+      <a href="https://www.qcif.edu.au" target="_blank">
+        <img src="{% static 'logos/footer/qcif-2.png' %}" />
+      </a>
+      <a href="https://www.aarnet.edu.au" target="_blank">
+        <img src="{% static 'logos/footer/aarnet.png' %}" />
+      </a>
+    </div>
+</div>
+
+<hr>
+
+<h3 id="computational-partners">
+  Computational Partners
+</h3>
+
+<p>
+Galaxy Australia is underpined by computational resources provided by
+  <a href="https://www.aarnet.edu.au/" target="_blank">AARNet</a>,
+  <a href="https://ardc.edu.au/services/ardc-nectar-research-cloud/" target="_blank">ARDC Nectar Research Cloud</a>,
+  <a href="https://www.unimelb.edu.au/" target="_blank">the University of Melbourne</a>,
   <a href="https://www.qcif.edu.au/" target="_blank">QCIF</a>,
   <a href="https://pawsey.org.au/" target="_blank">Pawsey Supercomputing Research Centre</a>,
   <a href="https://nci.org.au/" target="_blank">National Computational Infrastructure</a>,
-  <a href="https://www.aarnet.edu.au/" target="_blank">AARNet</a>,
-  <a href="https://ardc.edu.au/services/ardc-nectar-research-cloud/" target="_blank">ARDC Nectar Research Cloud</a>,
   and
   <a href="https://azure.microsoft.com/" target="_blank">Microsoft Azure</a>.
 </p>
+<div class="container logo">
+ <div class="text-center">
+   <a href="https://www.aarnet.edu.au" target="_blank">
+      <img src="{% static 'logos/footer/aarnet.png' %}" />
+   </a> 
+   <a href="https://ardc.edu.au/services/ardc-nectar-research-cloud/" target="_blank">
+      <img src="{% static 'logos/footer/ARDC_Nectar_Research_Cloud_RGB.png' %}" />
+   </a>
+   <a href="https://www.unimelb.edu.au" target="_blank">
+      <img src="{% static 'logos/footer/unimelb-2.svg' %}" />
+  </a>
+  <a href="https://www.qcif.edu.au" target="_blank">
+      <img src="{% static 'logos/footer/qcif-2.png' %}" />
+  </a>
+    <a href="https://nci.org.au" target="_blank">
+        <img src="{% static 'logos/footer/nci_logo.png' %}" />
+    </a>
+    <a href="https://pawsey.org.au" target="_blank">
+        <img src="{% static 'logos/footer/pawsey-grayscalelogo-beige.png' %}" />
+    </a>
+     <a href="https://azure.microsoft.com/en-au" target="_blank">
+        <img src="{% static 'logos/footer/Microsoft_Azure_Logo.svg.png' %}" />
+      </a>
+    </div>
+  </div>
```

Comment:
These look good after fixing a couple of typos in file names

![image](https://github.com/user-attachments/assets/78c0e41b-3239-49f8-a605-2d292a317071)

Reply:
Nice! Thanks for sorting those missing file paths out